### ISSUE TITLE
This error pops up

### ISSUE THREAD

**Issue by angelflesh** [chsiao58/EvenBetterArtisanGoodIcons#8]
Even Better Artisan Good Icons 1.6.6 because its DLL 'EvenBetterArtisanGoodIcons.dll' doesn't exist.

This is an error I keep getting in SMAPI and I am not sure how to fix it. I'd love to be able to use this mod so if you are able to walk me through the steps to fix this I would be grateful. Thank you!

**Comment by chsiao58** [comment]
Hmm, are you sure the DLL file is in the folder of the mod, and is it under the same name?

**Comment by Nycelios** [comment]
I am unfortunately getting the same issue I can't seem to find a DLL file in the folder either...

**Comment by empy1** [comment]
Same problem the DLL file is not there

**Comment by SpyroThyKing** [comment]
Getting the same error

**Comment by abandm** [comment]
This is true, I tried to copy the dll file from 1.6.5 and that didn't work either, though it didn't throw an error, just none of my wines are looking any different from the default.

the zip file looks a lot more like source code than the previous two version, so maybe the zip file is just incorrect and needs to be recreated

**Comment by DanDuhon** [comment]
I got this error when I cloned the repo, but not when I downloaded the latest release [here](https://github.com/chsiao58/EvenBetterArtisanGoodIcons/releases/tag/v1.6.6).  I know this is old-ish so maybe it was already fixed, but try that.

**Comment by chsiao58** [comment]
Wait, did you all just clone the repo? Please download release like DanDuhon said!

If you all fixed your issue after this then I will close this issue. I will make sure to emphasize this in the Read Me. Sorry about the unclarity.

**Comment by wafflebaffle** [comment]
Hello, I'm also getting this error and don't really know what to do. I downloaded the new release today without ever downloading prior versions and just unzipped the file as usual. Also can't find a ddl file. Thanks.

**Comment by DanDuhon** [comment]
So.. just confirming.  You go to the Releases page.
![image](https://github.com/user-attachments/assets/94d20255-a1aa-41cb-9b22-56c1a52900d5)

Then you download the zip file.
![image](https://github.com/user-attachments/assets/802670ed-08f4-484f-8048-f300a1bddead)

Open the .zip file and put the whole folder inside in your Mods folder.
![image](https://github.com/user-attachments/assets/37d5d4a5-9cee-422e-a5dc-0e94a6c0fea3)

If you look inside that folder, the ddl file is there.
![image](https://github.com/user-attachments/assets/d2e7411d-45f3-4413-9158-4b157c50e7bc)

**Comment by wafflebaffle** [comment]
I downloaded it again like normal and replaced the og folder and it worked so thank you, now I'm just weirded out as to why the first one wasn't working when it's the same zip, anyways thanks!!
